generated/proto/encodingpb/string.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+option go_package = "github.com/xichen2020/eventdb/generated/proto/encodingpb";
+
+import "github.com/xichen2020/eventdb/generated/proto/encodingpb/types.proto";
+
+package encodingpb;
+
+message StringMeta {
+    EncodingType encoding = 1;
+    CompressionType compression = 2;
+    // Byte packing for length in bits or dict encoded id.
+    int64 bytes_per_record = 3;
+}
+
+message StringMapping {
```

Comment:
`StringArray` might be a better name?

Reply:
Hm but this is just an array though, where dictionary usually represents a key value mapping.

Reply:
Okay, I'll change it `StringArray`.